README.md

```diff
@@ -0,0 +1,17 @@
+To copy configuration:
```

Comment:
no user will se this.

Reply:
It should be noted that reading this mean a distro package would have the same issues.